diagrams/lib/app_bar.dart

```diff
@@ -110,6 +110,7 @@ class AppBarDiagramStep extends DiagramStep {
 
   @override
   Future<List<File>> generateDiagrams() async {
+    controller.pixelRatio = 1.0 / 1.2;
```

Comment:
we should probably make this 1.2 a constant (and add a comment explaining what we're doing here).

Reply:
Done.